dockerfiles/Dockerfile.gunicorn

```diff
@@ -1,9 +1,11 @@
 # Dockerfile for running titiler application with gunicorn server
 # Size ~1.4GB
-ARG PYTHON_VERSION=3.9
+ARG PYTHON_VERSION=3.9-slim-2021-10-02
```

Comment:
If we're switching to slim I think it's safe to not go with the postfixed date version so that we always pick up the latest slim image.

```suggestion
ARG PYTHON_VERSION=3.9-slim
```

Reply:
FYI upgrading openssl etc had to be done for lots of repositories, see here https://github.com/20treeAI/sen2cor-pipelines/pull/11/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R16

Reply:
Further, if we can upgrade to python:3.11.0rc1-bullseye we fix another [18 vulnerabilities](https://app.snyk.io/org/roelof/project/f70adfc9-eefb-48b1-9b6b-904c93b97e2e)
![image](https://user-images.githubusercontent.com/10637696/191494173-f35324c7-480a-4c34-85c5-88c1e713883d.png)